src/main/java/io/stargate/sgv2/jsonapi/grpc/retries/impl/JsonApiGrpcRetryPolicy.java

```diff
@@ -0,0 +1,53 @@
+package io.stargate.sgv2.jsonapi.grpc.retries.impl;
+
+import io.grpc.Metadata;
+import io.grpc.Status;
+import io.grpc.StatusRuntimeException;
+import io.grpc.protobuf.ProtoUtils;
+import io.stargate.bridge.proto.QueryOuterClass;
+import io.stargate.sgv2.api.common.grpc.retries.GrpcRetryPredicate;
+import java.util.Objects;
+import javax.enterprise.context.ApplicationScoped;
+
+/** Default gRPC retry policy used in the project. */
```

Comment:
Maybe add a brief description of logic ("Retry UNAVAILABLE and true Server-side read/write timeouts")